README.md

```diff
@@ -346,7 +346,10 @@ Alacritty should work with the standard `xterm-256color` definition,
 but to allow programs to make best use of alacritty's capabilities,
 use its own terminfo definition instead.
 
-Note that the default configuration **requires** alacritty's terminfo.
+Note: unless the user has set the `TERM` environment variable in the
```

Comment:
I'd personally prefer it if the `Note: ` here would just be removed. Alternatively it should be made bold and the `unless` should be capitalized, but I think the text flow is better if this paragraph just stands as another normal line in the readme rather than a highlighted note.

Reply:
Removing `Note:` sounds good to me.